doc/source/user_guide/missing_data.rst

```diff
@@ -348,14 +348,21 @@ that, by default, performs linear interpolation at missing data points.
    np.random.seed(123456)
    idx = pd.date_range('1/1/2000', periods=100, freq='BM')
    ts = pd.Series(np.random.randn(100), index=idx)
-   ts[1:20] = np.nan
+   ts[1:5] = np.nan
+   ts[20:30] = np.nan
    ts[60:80] = np.nan
    ts = ts.cumsum()
 
 .. ipython:: python
 
    ts
    ts.count()
+   @savefig series_before_interpolate.png
+   ts.interpolate().plot()
```

Comment:
should this be without the interpolate? As the above line is already included a couple of lines below